### PR TITLE
feat(releasing): Create release candidate on automatic schedules

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
Move the release candidate creation from a manual action to a scheduled event.
The schedule is `at 15:00 UTC on Mondays, Wednesdays, Fridays`
We can only use UTC time for schedules
This action should create nothing when no additional PR are created.

### Motivation
Automation & efficiency

### CODE REVIEW

File: .github/workflows/create_rc_pr.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 name: Create RC PR
 
 on:
-  workflow_dispatch:
+  schedule:
+    - cron: '0 14 * * 1,3,5' # Run on Monday, Wednesday, and Friday at 16:00 UTC
```
Comment (CelianR) [inline .github/workflows/create_rc_pr.yml:5]: **❓ question:** ‏ Isn't it at 14:00 UTC ?

**Review by CelianR — APPROVED** [review]

File: .github/workflows/create_rc_pr.yml (hunk 1 above)
Comment (chouetz) [reply .github/workflows/create_rc_pr.yml:5]: yes I updated the code but not the comment...

**Pull request merged**